utils/to-valid-package-name.js

```diff
@@ -0,0 +1,21 @@
+const validate = require('validate-npm-package-name');
+
+module.exports = function toValidPackageName(packageName) {
+  const { validForNewPackages, validForOldPackages } = validate(packageName);
+  const isValid = validForNewPackages && validForOldPackages;
```

Comment:
Why only `validForNewPackages` check is not enough?

Reply:
Yes `validForNewPackages` is enough, thank you! `validForOldPackages` seems to be for legacy purpose only, and is more permissive, so that's not what we want! 

Fixed!